### PR TITLE
initial 1.3.27

### PR DESCRIPTION
**Pull request by M-Waszkiewicz-Anaconda**
s2n 1.3.27 

**Destination channel:** defaults

### Links

- [ticket_number](https://anaconda.atlassian.net/browse/PKG-3920) 
- [Upstream repository](https://github.com/aws/s2n-tls/tree/v1.3.27)

dependency for aws-c-io  0.13.10 -> aws-c-event-stream 0.2.15 -> aws-sdk-cpp 1.10.55 -> arrow-cpp 14.0.1 -> pyarrow 14.0.1 which would address CVE

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
 
 source:
   url: https://github.com/awslabs/{{ name }}/archive/v{{ version }}.tar.gz
-  sha256: 65d452d62119000c8056c569a0fe5bcbd261c464c41c945a8244df2e89855af8
+  sha256: 58dde9979867a3209a19098ffcd748a144b2462ec81edca5baf864313ff4f589
 
 build:
   number: 0
-  skip: true  # [win]
+  skip: true  # [win or s390x]
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
  skip: true  # [win]
```
We also have a request for pyarrow on s390x. Could this be enabled?

File: recipe/conda_build_config.yaml [hunk 2]
```diff
@@ -1,2 +1,8 @@
 MACOSX_SDK_VERSION:  # [osx and x86_64]
   - 10.12            # [osx and x86_64]
```
Comment (cbouss) [inline recipe/conda_build_config.yaml]: I see a mix and match of 10.12 and 10.13. Should this be aligned?

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
   run_exports:
     - {{ pin_subpackage("s2n", max_pin="x.x.x") }}
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
    - {{ pin_subpackage("s2n", max_pin="x.x.x") }}
  missing_dso_whitelist:
    - '$RPATH/ld64.so.1'  # [s390x]
```

**Review by cbouss — APPROVED** [review]

**Comment by Jrice1317** [comment]
This looks good and I understand wanting to address a CVE, but are we no longer creating versioned feedstocks when regressing the version of a feedstock?

**Comment by M-Waszkiewicz-Anaconda** [comment]
> This looks good and I understand wanting to address a CVE, but are we no longer creating versioned feedstocks when regressing the version of a feedstock?

This is the first Anaconda release, the diff shows what CF currently have

**Comment by Jrice1317** [comment]
> > This looks good and I understand wanting to address a CVE, but are we no longer creating versioned feedstocks when regressing the version of a feedstock?
> 
> This is the first Anaconda release, the diff shows what CF currently have

Ahhh, I see. I wonder if that's good for historic purposes. It's definitely confusing as a reviewer.

**Review by Jrice1317 — APPROVED** [review]
LGTM, but I don't think there should be a diff if it's an initial build.

**Comment by M-Waszkiewicz-Anaconda** [comment]
> LGTM, but I don't think there should be a diff if it's an initial build.

The fork was done much earlier, that's why

**Pull request merged**